### ISSUE TITLE
GOLDENGATE1 Heavy Swapping Reported During DB Creation

### ISSUE THREAD

**Issue by dfhawthorne** [dfhawthorne/oracle-db-build#21]
While executing the task, "Configure Oracle DB in Silent Mode", of the `orchestration/build_goldengate.yml` playbook, the following messages were seen in the DB alert log:

```
2024-01-04 21:27:28.778000 +11:00
Timed out trying to start process M000.
2024-01-04 21:30:57.446000 +11:00
Heavy swapping observed on system
2024-01-04 21:32:20.625000 +11:00
WARNING: Heavy swapping observed on system in last 5 mins.
2024-01-04 21:32:31.846000 +11:00
Heavy swapping can lead to timeouts, poor performance, and instance eviction.
```

**Comment by dfhawthorne** [comment]
`free -m` shows:

```
              total        used        free      shared  buff/cache   available
Mem:           4631        1541         198        2309        2891         551
Swap:          4135         257        3878
```

Paging activity is happening, but doesn't appear to be high. This could be masked by the high I/O load:

```
[oracle@goldengate1 ~]$ sar 15 5
Linux 4.18.0-147.el8.x86_64 (goldengate1.yaocm.id.au) 	04/01/24 	_x86_64_	(2 CPU)

21:58:41        CPU     %user     %nice   %system   %iowait    %steal     %idle
21:58:56        all      4.27      0.00      0.95     53.17      0.10     41.52
21:59:11        all     28.10      0.00      2.34     38.74      0.74     30.08
21:59:26        all     11.26      0.00      1.18     43.41      0.13     44.01
21:59:41        all      1.91      0.00      1.54     57.48      0.97     38.10
21:59:56        all     16.20      0.00      2.36     50.62      0.17     30.65
Average:        all     12.37      0.00      1.68     48.67      0.42     36.86
```

The I/O load is confirmed by `iostat`:

```
[oracle@goldengate1 ~]$ iostat 15 5
Linux 4.18.0-147.el8.x86_64 (goldengate1.yaocm.id.au) 	04/01/24 	_x86_64_	(2 CPU)

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           2.18    0.08    1.26   53.87    0.52   42.10

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
xvda             47.41      1715.73      1880.89   61751536   67695798
dm-0             48.83      1709.43      1877.13   61524768   67560707
dm-1              2.83         2.60         9.20      93496     330984

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.88    0.00    1.01   54.07    0.13   40.90

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
xvda             29.80       939.20      1251.10      14088      18766
dm-0             29.93       942.67      1251.80      14140      18777
dm-1              0.00         0.00         0.00          0          0

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           4.68    0.00    1.04   47.73    0.34   46.21

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
xvda             14.67        21.87       205.57        328       3083
dm-0             14.73        17.07       206.53        256       3098
dm-1              0.27         1.07         0.00         16          0

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           1.72    0.00    0.74   51.43    0.10   46.00

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
xvda             11.13         8.53        97.07        128       1456
dm-0             11.07         8.53        96.80        128       1452
dm-1              0.00         0.00         0.00          0          0

avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           6.80    0.00    1.34   46.25    0.33   45.28

Device             tps    kB_read/s    kB_wrtn/s    kB_read    kB_wrtn
xvda             19.06        48.61       553.42        732       8334
dm-0             17.66        42.50       551.96        640       8312
dm-1              2.12         6.37         2.12         96         32
```

**Comment by dfhawthorne** [comment]
Fixed by commit [dfe8969](https://github.com/dfhawthorne/xen_server_farm/commit/dfe8969fece5e34debd0a6fc6b0b730931752cdf) in the [xen_server_farm](https://github.com/dfhawthorne/xen_server_farm) repository.
